FirebaseRemoteConfig/Sources/RCNConfigContent.m

```diff
@@ -134,6 +140,14 @@ - (void)loadConfigFromMainTable {
     self->_activePersonalization = [activePersonalization copy];
     dispatch_group_leave(self->_dispatch_group);
   }];
+
+  dispatch_group_enter(_dispatch_group);
```

Comment:
Is it possible to do this in `loadMainWithBundleIdentifier`? It looks like there's an existing todo above to do that for personalization, so it'd be nice if we can avoid it here